jessie_update/jessie-heavy-upgrade.sh

```diff
@@ -17,27 +17,46 @@
 ###################################################
 
 ###################################################
+# remove all extraneous packages from Jessie 
+# trying to gain as much space as possible
+# change CleoQc to DexterInd when doing PR
+###################################################
+rm ./jessie_cleanup.sh  # just making sure there isn't one yet
+wget https://raw.githubusercontent.com/CleoQc/Raspbian_For_Robots/master/jessie_update/jessie_cleanup.sh
+chmod +x ./jessie_cleanup.sh
+./jessie_cleanup.sh
+rm ./jessie_cleanup.sh
+
+###################################################
+# give Raspbian its flavor. After this, host will be dex, and pi password will be robots1234
+# Samba will also be installed with password set to robots1234
+###################################################
+rm ./Raspbian_for_Robots_Flavor.sh
+wget https://raw.githubusercontent.com/CleoQc/Raspbian_For_Robots/master/jessie_update/Raspbian_for_Robots_Flavor.sh
+chmod +x ./Raspbian_for_Robots_Flavor.sh
+./Raspbian_for_Robots_Flavor.sh
+rm ./Raspbian_for_Robots_Flavor.sh
+
+###################################################
 ##
-## Take Jessie Lite to Raspbian
+## Take "Jessie on a Diet" to Raspbian
 ##
 ###################################################
 
 sudo apt-get update -y
 sudo apt-get upgrade -y
 
+# is dist-upgrade necessary ? 
 sudo apt-get dist-upgrade -y
+
+# is apache2 necessary? 
```

Comment:
Yes, this is the server that dex.local homepage is running on.  Plus other dependencies.